httpclient5/src/main/java/org/apache/hc/client5/http/impl/io/PoolingHttpClientConnectionManagerBuilder.java

```diff
@@ -70,10 +74,13 @@
  */
 public class PoolingHttpClientConnectionManagerBuilder {
 
+    private Lookup<ConnectionSocketFactory> connectionSocketFactoryRegistry;
     private HttpConnectionFactory<ManagedHttpClientConnection> connectionFactory;
     private LayeredConnectionSocketFactory sslSocketFactory;
     private SchemePortResolver schemePortResolver;
     private DnsResolver dnsResolver;
+
+    private StrictConnPool<HttpRoute, ManagedHttpClientConnection> managedConnPool;
```

Comment:
@rafiek Why do we need this variable? It invalidates `PoolConcurrencyPolicy` parameter.

Reply:
It is used in tests, so I added the option to make it part of the `build()`. When omitted, you can still use `PoolConcurrencyPolicy`.